website/docs/enforce/enforce_rules.md

```diff
@@ -57,8 +57,11 @@ Enforce rules are functions that allow you to test your data against different c
 - [doesNotStartWith](#doesnotstartwith)
 - [isNegative](#isnegative)
 - [isPositive](#ispositive)
+- [isKeyOf](#iskeyof)
+- # [isNotKeyOf](#isnotkeyof)
 - [isValueOf](#isvalueof)
 - [isNotValueOf](#isnotvalueof)
+  > > > > > > > origin/latest:website/docs/enforce/enforce_rules.md
```

Comment:
Looks like you had a tiny merge conflict here, let's fix it and we're good to go
```suggestion
- [isKeyOf](#iskeyof)
- [isNotKeyOf](#isnotkeyof)
- [isValueOf](#isvalueof)
- [isNotValueOf](#isnotvalueof)
```